packages/icon-button/src/IconButton.story.tsx

```diff
@@ -25,16 +26,18 @@ export default {
 };
 // eslint-disable-next-line react/prop-types
 const Template: ComponentStory<typeof IconButton> = ({ darkMode, ...args }) => (
-  <div
-    className={css`
-      padding: 60px;
-      ${darkMode && `background-color: ${palette.gray.dark3};`}
-    `}
-  >
-    <IconButton darkMode={darkMode} {...args}>
-      <CloudIcon />
-    </IconButton>
-  </div>
+  <LeafyGreenProvider>
+    <div
+      className={css`
+        padding: 60px;
+        ${darkMode && `background-color: ${palette.gray.dark3};`}
+      `}
+    >
+      <IconButton darkMode={darkMode} {...args}>
+        <CloudIcon />
+      </IconButton>
+    </div>
+  </LeafyGreenProvider>
```

Comment:
I've added `LeafygreenProvider` globally as a Storybook decorator, so we shouldn't need components wrapped with it explicitly in stories any more!

Reply:
Same goes for the div with the darkMode styles - I've made `DarkModeWrapper` a decorator that is applied globally.